1.23/base/Dockerfile

```diff
@@ -26,8 +26,8 @@ RUN \
             yamllint \
             openssh-client \
         && git config --system --add safe.directory "*" \
-        && curl -s -f -L https://deb.nodesource.com/gpgkey/nodesource.gpg.key -o /etc/apt/trusted.gpg.d/nodesource.asc \
-        && echo "deb https://deb.nodesource.com/node_20.x/ bookworm main" > /etc/apt/sources.list.d/nodesource.list \
+        && curl -fsSL https://deb.nodesource.com/setup_22.x -o nodesource_setup.sh \
```

Comment:
I would much prefer to not use the script setup like this. Is there a reason the previous `apt` method doesn't work anymore?

Reply:
yes that's because it doesn't exist unfortunately :(